### PR TITLE
[BD-05][TNL-7311] Add ORA aggregated data API - WIP

### PR DESCRIPTION
**Pull request by giovannicimolin**
**TL;DR -** This PR adds internal python APIs and endpoints to be used by the upcoming ORA instructor dashboard rework.

**JIRA Tickets:**
- [TNL-7311](https://openedx.atlassian.net/browse/TNL-7311)

**What changed?**

WIP

**Testing instructions:**

1. Checkout this branch.
2. Add this to your `lms/urls.py`:
```
# ORA API views
urlpatterns += [
    url(r'^api/v1/openassessment/', include('openassessment.urls')),
]
```
3. Restart LMS.
4. TBD

**Developer Checklist**
- [ ] Reviewed the [release process](./release_process.md)
- [ ] Translations up to date
- [ ] JS minified, SASS compiled
- [ ] Test suites passing on Jenkins
- [ ] Bumped version number in [setup.py](../setup.py) and [package.json](../package.json)

FIY: @edx/masters-devs-gta

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @giovannicimolin! I've created [BLENDED-410](https://openedx.atlassian.net/browse/BLENDED-410) to keep track of it in Jira.

When this pull request is ready, tag your edX technical lead.
